versions.json

```diff
@@ -1,4 +1,69 @@
 {
+  "21": {
+    "start": "2023-10-17",
+    "lts": "",
+    "maintenance": "2024-04-01",
+    "end": "2024-06-01",
+    "codename": "",
+    "alpine-default": "alpine3.18",
+    "debian-default": "bookworm",
+    "variants": {
+      "alpine3.17": [
+        "amd64",
+        "arm32v6",
+        "arm32v7",
+        "arm64v8",
+        "ppc64le",
+        "s390x"
+      ],
+      "alpine3.18": [
+        "amd64",
+        "arm32v6",
+        "arm32v7",
+        "arm64v8",
+        "ppc64le",
+        "s390x"
+      ],
+      "bookworm": [
+        "amd64",
+        "arm32v7",
+        "arm64v8",
+        "ppc64le",
+        "s390x"
+      ],
+      "bookworm-slim": [
+        "amd64",
+        "arm32v7",
+        "arm64v8",
+        "ppc64le",
+        "s390x"
+      ],
+      "bullseye": [
+        "amd64",
+        "arm32v7",
+        "arm64v8",
+        "ppc64le",
+        "s390x"
+      ],
+      "bullseye-slim": [
+        "amd64",
+        "arm32v7",
+        "arm64v8",
+        "ppc64le",
+        "s390x"
+      ],
+      "buster": [
```

Comment:
We should probably remove buster. But I guess 21 itself will be EOL at about the same time buster is, so no rush

Reply:
Yeah I think that's a fair point. You probably don't want to create a buster variant when Node v22 is released, given that buster will almost be EOL by that point. So not creating the variant now saves some build time for the official image builder (they're probably happy about that) and encourages people to migrate to the newer Debian versions. Thus I've removed it.